bay_trimesh/anaconda-project.yml

```
@@ -2,37 +2,37 @@
 name: bay_trimesh
 description: Visualizing water depth into the Chesapeake and Delaware Bays
 maintainers:
-  - philippjfr
+- philippjfr
 labels:
-  - geoviews
-  - datashader
+- geoviews
+- datashader
 
-user_fields: [labels, skip, maintainers]
+user_fields: [labels, skip, maintainers, user_fields, user_fields, user_fields, user_fields,
+  user_fields, user_fields]
```

```suggestion
user_fields: [labels, skip, maintainers]
```

Is there a bug in anaconda-project that is creating all those bogus copies of `user_fields` in this list?

---

I believe there is, I'll go through and clean up all the extra `user_fields` entries.